dts/bindings/spi/spi-device.yaml

```diff
@@ -16,3 +16,15 @@ properties:
       description: Maximum clock frequency of device's SPI interface in Hz
     label:
       required: true
+    duplex:
+      type: int
+      required: false
+      description: Duplex mode, full or half. By default it's always full
```

Comment:
nit for multiline description: use `description: |`

Reply:
Seconded

Reply:
only if you break the line right after description:  or for all? 

I saw confusing examples in other bindings. Anyway I'll break the line.

Reply:
> only if you break the line right after description: or for all?

The rules and rationale for them are documented in 
https://docs.zephyrproject.org/latest/guides/dts/bindings.html#general-rules